corehq/apps/reports/views.py

```diff
@@ -162,6 +164,20 @@
 require_can_view_all_reports = require_permission(HqPermissions.view_reports)
 
 
+def _augmented_form_view_permissions():
+    def decorator(view_func):
+        @wraps(view_func)
+        def _inner(request, domain, *args, **kwargs):
+            if VIEW_FORM_ATTACHMENT.enabled(domain):
+                return view_func(request, domain, *args, **kwargs)
```

Comment:
Guessing this should still used `login_and_domain_required`, otherwise the views would essentially be public.  Actually I guess that view also uses `@api_auth`, so I suppose it is protected via that.  Still, seems safer that way if it's to be used elsewhere, as it'd be easy to forget coupling it with another decorator, since typically these permissions decorators ensure domain access in addition to the required permission.

Reply:
@esoergel Oh, I see what you mean! Thanks for flagging this!